src/operations/distortion.jl

```diff
@@ -93,7 +93,7 @@ augment(img, ElasticDistortion(15, 15; scale = 0.1))
 # distort everything more smoothly.
 augment(img, ElasticDistortion(10, 10; sigma = 4, iter=3, border=true))
 ```
-""" ->
+"""
```

Comment:
are you sure?

Reply:
Ah, these two changes are made in 8dbe2b1 (#27 ) and my changes #31 is built on top of that.

I haven't observe such `->` usage after Julia 1.0; so I guess it's an deprecated syntax